rsocket/DuplexConnection.h

```diff
@@ -29,9 +54,9 @@ namespace rsocket {
 /// before the connection is destroyed.
 class DuplexConnection {
  public:
-  using InternalSubscriber =
-      yarpl::flowable::InternalSubscriber<std::unique_ptr<folly::IOBuf>>;
-  using Subscriber = yarpl::flowable::Subscriber<std::unique_ptr<folly::IOBuf>>;
+  using Subscriber =
+      yarpl::flowable::Subscriber<std::unique_ptr<folly::IOBuf>>;
+  using DuplexSubscriber = rsocket::DuplexSubscriber;
```

Comment:
can we still call it DuplexConnection::Subscriber? I think DuplexSubscriber is confusing..

Reply:
I don't think there's a clean way to do that, the DuplexConnection deals with the generic Subscriber interface in setInput/getOutput, and DuplexSubscriber is the old `InternalSubscriber` analog. It just so happened that the Subscriber interface previously implemented InternalSubscriber.